Flowchart/Views/Home/Contact.cshtml

```diff
@@ -1,15 +1,21 @@
 ﻿@{
     ViewData["Title"] = "Contact";
 }
+<head>
+    <style>
+
+    </style>
+</head>
 <h2>@ViewData["Title"].</h2>
+<p>feel free to contact us for assistance</p>
 <dl>
-   <dt>email Sam Messina:</dt>
+   <dt><font size="5">email Sam Messina:</font></dt>
```

Comment:
In fact, the font tag[ is not supported in HTML5](https://www.w3schools.com/tags/tag_font.asp)